xml/System.Net/HttpWebRequest.xml

```diff
@@ -102,6 +102,15 @@
       </AssemblyInfo>
       <Docs>
         <summary>Initializes a new instance of the <see cref="T:System.Net.HttpWebRequest" /> class.</summary>
```

Comment:
Should we add also to the summary something like: "Obsoleted, use Create - see remarks for details."?
I would think that we want to convey the obsoleted information ASAP. Remarks are fairly low on the page and might be easily missed IMO.